webapp/playbooks/tsconfig.json

```diff
@@ -25,7 +25,7 @@
     "paths": {
       "src": ["src"],
       "mattermost-redux/*": ["../channels/src/packages/mattermost-redux/src/*"],
-      "reselect": ["../channels/src/packages/reselect/src"]
+      "packages/reselect/src" : ["../channels/src/packages/reselect/src"],
```

Comment:
this is needed for ts compiler of Playbooks to correctly resolve when mattermost-redux is imported in playbooks code